src/markdown.ts

```diff
@@ -176,8 +176,8 @@ function formatName(name = "") {
     .join(" ");
 }
 
-function groupBy(items: any[], key: string) {
-  const groups = {};
+function groupBy<TItem>(items: TItem[], key: string) {
+  const groups: Record<string, TItem[]> = {};
```

Comment:
This function was returning `any` for me locally, seemed useful to provide types